HeterogeneousCore/AlpakaTest/plugins/TestAlpakaAnalyzer.cc

```diff
@@ -134,19 +139,32 @@ class TestAlpakaAnalyzer : public edm::global::EDAnalyzer<> {
       assert(vi.id() == i);
       assert(vi.m() == matrix * i);
     }
+
+    if (expectBackend_) {
+      auto const& backend = event.get(backendToken_);
+      if (*expectBackend_ != backend) {
```

Comment:
```suggestion
      if (expectBackend_ != backend) {
```
the comparison should also work without the explicit indirection.

Reply:
Thanks.